firewall-examples.md

```diff
@@ -174,48 +167,41 @@ $ sdc-createfirewallrule --enabled --rule "FROM any TO tag role = www ALLOW tcp
 </pre>
 </div></div>
 
-<p>After you have created both of these rules, instances with the tag <tt>role</tt> set to <tt>db</tt> will have the following behavior:</p>
-
-<ul>
-	<li>Allow incoming TCP traffic on port 5432 from instances with tag <tt>role=www</tt></li>
-	<li>Allow all outgoing traffic</li>
-	<li>Allow incoming ping requests</li>
-	<li>Block all other incoming traffic</li>
-</ul>
+After you have created both of these rules, instances with the tag `role` set to `db` will have the following behavior:
 
+	- Allow incoming TCP traffic on port 5432 from instances with tag `role=www`
+	- Allow all outgoing traffic
+	- Allow incoming ping requests
+	- Block all other incoming traffic
 
-<p>And instances with the tag <tt>role</tt> set to <tt>www</tt> will have the following behavior:</p>
 
-<ul>
-	<li>Allow incoming TCP traffic on ports 80 and 443 from any IP address</li>
-	<li>Allow outgoing TCP traffic on port 5432 to instances with  <tt>tag role=www</tt></li>
-	<li>Allow all outgoing traffic</li>
-	<li>Allow incoming ping requests</li>
-	<li>Block all other incoming traffic</li>
-</ul>
+And instances with the tag `role` set to `www` will have the following behavior:
 
+	- Allow incoming TCP traffic on ports 80 and 443 from any IP address
+	- Allow outgoing TCP traffic on port 5432 to instances with  `tag role=www`
+	- Allow all outgoing traffic
+	- Allow incoming ping requests
+	- Block all other incoming traffic
 
-<p>Creating additional instances with the role tags listed above will automatically<br/>
-apply these rules. For example, to apply the web server rules to a new server,<br/>
-just give it tag <tt>role=www</tt>.</p>
+Creating additional instances with the role tags listed above will automatically   
+apply these rules. For example, to apply the web server rules to a new server,   
```

Comment:
One of the things we'll want to do is eliminate the hard wrapping in the existing docs and let paragraphs continue unbroken.